ansible/roles/service/base_image/defaults/main.yml

```diff
@@ -1,5 +1,6 @@
 ---
 
 base_image:
+  sysstat_enabled: "false"
```

Comment:
I don't know much about sysstat. Why do we want to disable it?

Reply:
no need to collect CPU, memory, network usage by default. We can turn it on/off in case we need to - like for load testing etc.